packages/mom5/package.py

```diff
@@ -488,6 +496,10 @@ def build(self, spec, prefix):
             build = Executable("./MOM_compile.csh")
             if "+restart_repro" in self.spec:
                 build.add_default_env("REPRO", "true")
+
+            if self.spec.satisfies("+access-gtracers"):
+                build.add_default_env("SPACK_BUILD", "true")
```

Comment:
Just a note for self, the corresponding change in MOM5 is 

https://github.com/mom-ocean/MOM5/commit/a768276fc76f07ef09cfec3cfabb6feef92667d1